workflow/commons/src/test/java/com/datastax/oss/dsbulk/workflow/commons/settings/SchemaSettingsTest.java

```diff
@@ -1169,7 +1169,7 @@ void should_create_single_read_statement_when_no_variables() {
   void should_create_multiple_read_statements() {
     ColumnDefinitions definitions =
         mockColumnDefinitions(
-            mockColumnDefinition("start", BIGINT), mockColumnDefinition("end", BIGINT));
```

Comment:
Is this change to fix some test failure?

Reply:
yes.